### ISSUE TITLE
MJML, HTML code and rendered display should be included side by side

### ISSUE THREAD

**Issue by rootwork** [premail/website#3]
e.g. for https://premail.dev/docs/components/mjml-bullet-list/

the rendered display could just be an image

Docusaurus supports this with [tabs](https://docusaurus.io/docs/markdown-features/tabs).

**Comment by rootwork** [comment]
Done!
